money_honey_cr.py

```diff
@@ -0,0 +1,147 @@
+import json
+import sys
+import getopt
+from decimal import Decimal, InvalidOperation
+from random import randint
+
+from actual_currencies import CurrenciesChanger
+
+
+class MoneyHoney:
+    """
+    I wasn't sure if it's allowed to use external libraries so I didn't use any.
+    Estimated time of coding 4MD.
+    """
+    def __init__(self):
+        pass
```

Comment:
Not sure why I left it there. Removed in [#6](https://github.com/ZlatyChlapec/MoneyHoney/commit/f69f0f6253e8de5851e05a24c577f10b1f4c90ae).